cohortextractor/mssql_utils.py

```diff
@@ -93,6 +102,24 @@ def _ctds_connect(ctds, params):
     return ctds.connect(**params)
 
 
+def stats_msg_handler(msgstate, severity, srvname, procname, line, msgtext):
+    """
+    Log parse, compile and execution timing messages sent by the server.
+    """
+    if timing_match := SQLSERVER_TIMING_REGEX.match(msgtext.decode()):
```

Comment:
![image](https://user-images.githubusercontent.com/1042/173086548-f15b410c-94d3-4e34-8c92-79fc9908c5c9.png)

Reply:
First time actually using it :)